apps/mobile/src/screens/onboarding/onboarding.tsx

```diff
@@ -161,15 +159,15 @@ const Onboarding: React.FC = () => {
 								>
 									<InfinitePager
 										infinitePagerReference={infinitePager}
-										onPageRender={renderPageComponent}
+										onPageRender={handleRenderPageComponent}
 									/>
 									<View style={globalStyles.gap16}>
 										<Button
 											isDisabled={!isValid}
 											label={isLastQuestion ? "ANALYZE" : "NEXT"}
 											onPress={handleFormSubmit}
 										/>
-										{currentQuestionIndex !== ZERO && (
+										{currentQuestionIndex !== NumericalValue.ZERO && (
```

Comment:
TBH I think it is still not very clear. I would create a constant `FIRST_ITEM_INDEX` or smth like that

Reply:
Do I need to rename const ZERO_INDEX = 0; in shared or in component create const FIRST_ITEM_INDEX?

Reply:
Let's create `FIRST_ITEM_INDEX ` in component